core/src/io/anuke/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -357,4 +357,9 @@ public static void insulators(int x, int y, int x2, int y2, Cons<Tile> iterator)
             return false;
         });
     }
+
+    @Override
+    public void drawBottleneck(Tile tile){
```

Comment:
This wouldn't be needed if `drawBottleneck` only triggered for blocks with any consumers.